### ISSUE TITLE
[BUG] aura and scaffold rotations

### ISSUE THREAD

**Issue by MikosLeftThigh** [CCBlueX/LiquidBounce#1593]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

git-e572483

### Operating System

Windows 11

### Minecraft Version

1.20.1

### Describe the bug

after the new rotation "improvement" (version d003b22) the killaura and scaffold rots are a little bit broken, missing the enemy's hitbox all of the time regardless of the configuration.  
![image](https://github.com/CCBlueX/LiquidBounce/assets/143325613/882ef2ef-227c-4600-804a-1b5658409b94)
oh and btw raycast does nothing at the moment, you still flag hitbox - reach.

i suggest to either revert these changes or to rework them.

### Steps to reproduce

.

### Client Log

```shell
.
```

**Comment by 1zun4** [comment]
I have not noticed these issues myself, but I will look into it.

**Comment by 1zun4** [comment]
https://github.com/CCBlueX/LiquidBounce/assets/12410754/96fde57f-4640-49d3-a4c8-3d16fd146bf9
https://github.com/CCBlueX/LiquidBounce/assets/12410754/1f59daec-76a2-49c3-9eda-1ca418097e0c

I was not able to confirm your issue.

**Comment by 1zun4** [comment]
Try now: https://github.com/CCBlueX/LiquidBounce/commit/7a02ffed02070d8894db1091a503f5bf26fc8932
This basically reverts the changes from @mems01 that used client-side rotation variable (which should not be that much of an issue, as we are pre-aiming - but maybe it is): https://github.com/CCBlueX/LiquidBounce/commit/d374d58f20b42b0a1bf15bdc8be3d54763e60545

**Comment by MikosLeftThigh** [comment]
> > > https://github.com/CCBlueX/LiquidBounce/assets/12410754/96fde57f-4640-49d3-a4c8-3d16fd146bf9 https://github.com/CCBlueX/LiquidBounce/assets/12410754/1f59daec-76a2-49c3-9eda-1ca418097e0c
> > > I was not able to confirm your issue. Also raycast does not do anything to your reach configuration. Raycast is to make sure you do not hit through another enemy.
> > 
> > 
> > what is your aura config? mine flags simulation with only the settings shown in the first clip *well... turns out viafabric was making me flag, nice 👍
> 
> Can you please tell me what settings you were using on ViaFabric? 1.8 Protocol? That would help a lot to find the issue.

yes it was 1.8 protocol, sorry for replying late, i forgot to check github

**Comment by mems01** [comment]
ViaFabricPlus got updated recently. I cannot reproduce the issue, therefore I'm closing this. If you still somehow notice it still exists, feel free to reopen.
